concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md

```diff
@@ -24,4 +24,4 @@ The following code uses SVG icons in the [Button](https://js.devexpress.com/Demo
             icon: myIcon
         });
 
-[important]The SVG format allows running executable code that might be malicious. We strongly recommend that you use SVG icons only from trusted sources.
+[important]The SVG format allows to run executable code that might be malicious. We strongly recommend that you use SVG icons only from trusted sources.
```

Comment:
It's redundant to say both "run" and "executable". If it's executable, you can run it. If you can run it, it's executable. 

```suggestion
[important]The SVG format can contain executable code that might be malicious. We strongly recommend that you use SVG icons only from trusted sources.
```